.github/workflows/per-pr.yml

```diff
@@ -37,7 +37,7 @@ jobs:
       - name: Run flag test
         run: ${{ github.workspace }}/scripts/test-proxy-startup-with-env-vars.sh
 
-  test-local:
+  unit-tests:
     outputs:
       COVERAGE: ${{ steps.unit.outputs.coverage }}
```

Comment:
this can be removed

Reply:
it doesnt actually do anything, plus we don't even have a unit step anymore.

Reply:
[fe4424a](https://github.com/Layr-Labs/eigenda-proxy/pull/337/commits/fe4424ab371197957a52034eb4e3f9a812884d59)